src/index.html

```diff
@@ -333,11 +333,12 @@ <h3 class="header" itemprop="name">Can I save the certificate on Android?</h3>
                           <div class="content" itemscope itemprop="mainEntity" itemtype="https://schema.org/Question">
                               <h3 class="header" itemprop="name">How can I get my EU&nbsp;DCC (certificate)?</h3>
                               <div class="description" itemscope itemprop="acceptedAnswer" itemtype="https://schema.org/Answer">
+                                <p itemprop="text"><span aria-hidden="true">&#x1F1E7;&#x1F1EA;</span> In Belgium, you can download your certificate from (French), <a href="https://mijngezondheid.be">Mijngezondheid.be</a> (Dutch) or <a href="https://meinegesundheit.be">Meinegesundheit.be</a> (German).</p>
                                 <p itemprop="text"><span aria-hidden="true">&#x1F1E7;&#x1F1EC;</span> In Bulgaria, the certificate can be downloaded from the <a href="https://www.his.bg/bg/dgc">Ministry of Health's website</a></p>
-                                <p itemprop="text"><span aria-hidden="true">&#X1F1E8;&#X1F1FF;</span> In Czechia, you get your certificate from Ministry of Health <a href="https://ocko.uzis.cz" title="Očkovací portál občana">Vaccination portal</a> or via <i>Tečka</i> app.</p>
+                                <p itemprop="text"><span aria-hidden="true">&#x1F1E8;&#x1F1FF;</span> In Czechia, you get your certificate from Ministry of Health <a href="https://ocko.uzis.cz" title="Očkovací portál občana">Vaccination portal</a> or via <i>Tečka</i> app.</p>
                                 <p itemprop="text"><span aria-hidden="true">&#x1F1E9;&#x1F1F0;</span> In Denmark, you can get it in the Coronapas mobile app, when you select the travel option.</p>
                                 <p itemprop="text"><span aria-hidden="true">&#x1F1EB;&#x1F1F7;</span> In France, you can download it from <a href="https://attestation-vaccin.ameli.fr/" title="Visiter le site Ameli.fr pour télécharger votre certificat">Ameli’s dedicated website</a>.</p>
-                                <p itemprop="text"><span aria-hidden="true">&#x1F1EC;&#x1F1F7;</span> In Greece, you can download it from <a href="https://eudcc.gov.gr/" title="Επισκεφθείτε το gov.gr για την έκδοση Ευρωπαϊκού ψηφιακού πιστοποιητικού COVID-19">gov.gr</a> using your TaxisNet credentials.</p>
+                                <p itemprop="text"><span aria-hidden="true">&#x1F1EC;&#x1F1F7;</span> In Greece, you can download your certificate from <a href="https://dilosi.services.gov.gr/login?template=COVID19-VACCINE-2&next=/templates/COVID19-VACCINE-2/create&" title="Επισκεφθείτε το gov.gr για την έκδοση Ευρωπαϊκού ψηφιακού πιστοποιητικού COVID-19">Gov.gr</a> or from <a href="https://www.gsis.gr/en/citizens-businesses/remote-service/mykeplive">myKEPlive</a>.</p>
```

Comment:
For that one, I prefer the community version to use eudcc.gov.gr. 
DoctorAnyTime asked me this specific instruction for their version, but the URL in the community edition asks you a few simple questions and guides you to the correct system, depending on your situation.

Maybe we should remove "using your TaxisNet credentials because it depends then on which system you are sent to.

Reply:
My thoughts exactly. Here you go…